src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,43 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div className="card-image">
+      <figure
+        className="image is-4by3"
+      >
+        <img
+          alt="Film logo"
+          src={movie.imgUrl}
```

Comment:
The alt attribute of the img tag is hardcoded as 'Film logo'. It would be more accessible to use the movie title for the alt text.